### PR TITLE
Update yaml files for che6

### PR DESCRIPTION
**Pull request by davidfestal**
Update yaml files for running OSIO on Che 6

### CODE REVIEW

File: openshift/rh-che.app.yaml [hunk 1]
```diff
@@ -74,40 +74,15 @@ objects:
...
-            valueFrom:
-              configMapKeyRef:
-                key: workspace-storage-create-folders
-                name: che
           - name: CHE_LOCAL_CONF_DIR
             valueFrom:
               configMapKeyRef:
                 key: local-conf-dir
                 name: che
-          - name: CHE_OPENSHIFT_PROJECT
```
Comment (garagatyi) [inline openshift/rh-che.app.yaml]: I think we can inline value instead of setting it in config map since we won't use any other value except `openshift`

Comment (garagatyi) [reply openshift/rh-che.app.yaml]: sorry, wrong line. I was thinking about CHE_INFRASTRUCTURE_ACTIVE

Comment (davidfestal) [reply openshift/rh-che.app.yaml]: yes, I thought about that also. @l0rd @ibuziuk wdyt ? should we still use a config map value for this ?

Comment (ibuziuk) [reply openshift/rh-che.app.yaml]: +1 to inline CHE_INFRASTRUCTURE_ACTIVE - less config we have the better

Comment (davidfestal) [reply openshift/rh-che.app.yaml]: done in commit https://github.com/redhat-developer/rh-che/pull/553/commits/cd863556f3729f1c0f150130a7cf23407f114aa8

**Pull request merged**